CHANGELOG.md

```diff
@@ -1,247 +1,515 @@
 # Change Log
 
+## v1.8.0
+
+> Change log since v1.7.3
+
+### Upgrade Notice
+
+> No, really, you must read this before you upgrade
+
+- **Disable** the following feature gates by default:
+  ResourcesDeletionProtection ([#1919](https://github.com/openkruise/kruise/pull/1919), [@ABNER-1](https://github.com/ABNER-1))
+- Promote these feature gates to beta:  
+  `ResourcesDeletionProtection`, `WorkloadSpread`, `PodUnavailableBudgetDeleteGate`, `InPlaceUpdateEnvFromMetadata`,
+  `StatefulSetAutoDeletePVC`,
+  `PodProbeMarkerGate` ([#1919](https://github.com/openkruise/kruise/pull/1919), [@ABNER-1](https://github.com/ABNER-1))
+- Update Kubernetes dependency to v1.30.9 and Golang to
+  v1.22 ([#1896](https://github.com/openkruise/kruise/pull/1896), [@ABNER-1](https://github.com/ABNER-1))
+
+### Key Features
+
+- Support StatefulSet volume
+  expansion ([#1674](https://github.com/openkruise/kruise/pull/1674), [#1714](https://github.com/openkruise/kruise/pull/1714), [@ABNER-1](https://github.com/ABNER-1))
+- Support in-place resource
+  resizing ([#1353](https://github.com/openkruise/kruise/pull/1353), [#1866](https://github.com/openkruise/kruise/pull/1866), [@LavenderQAQ](https://github.com/LavenderQAQ), [@ABNER-1](https://github.com/ABNER-1))
+
+- Add Helm pre-delete hook to preserve Kruise CRs during
+  uninstallation ([#1843](https://github.com/openkruise/kruise/pull/1843), [@AiRanthem](https://github.com/AiRanthem))
+
+### Performance Improvements
+
+- Optimize CA bundle updates to reduce unnecessary
+  changes ([#1717](https://github.com/openkruise/kruise/pull/1717), [@zmberg](https://github.com/zmberg))
+- Add disableDeepCopy for
+  BroadcastJob ([#1696](https://github.com/openkruise/kruise/pull/1696), [@Prepmachine4](https://github.com/Prepmachine4))
+
+### Resilience Enhancement
+
+- Extend PodProbeMarker to serverless
+  pods ([#1875](https://github.com/openkruise/kruise/pull/1875), [@zmberg](https://github.com/zmberg))
+- Support adaptive scheduling strategy for
+  UnitedDeployment ([#1720](https://github.com/openkruise/kruise/pull/1720),  [@AiRanthem](https://github.com/AiRanthem))
```

Comment:
these two enhancements is not for Resilience, consider move the "Support adaptive scheduling strategy for
  UnitedDeployment" to "key features" section, and "Extend PodProbeMarker to serverless
  pods" to "Other Notable Changes"